### PR TITLE
kelly/cashier

### PR DESCRIPTION
**Pull request by kelly-binary**
Add Deposit and Withdrawal page + tidy up.

### CODE REVIEW

**Review by mohammad-deriv — APPROVED** [review]

File: src/templates/cashier/deposit_withdraw.html.tt [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        <div class="row">
+            <div id="error_msg" class="col-xs-12 notice-msg center-text hidden"></div>
+            <div id="ukgc_funds_protection" class="hidden">
+                <div class="col-xs-12">
+                    <p>
+                        [% l("We are required by our license to inform you about what happens to funds which we hold on account for you, and the extent to which funds are protected in the event of insolvency <a href='[_1]' target='_blank'>[_1]</a>.", "http://www.gamblingcommission.gov.uk/Consumers/Protection-of-customer-funds.aspx")
+                        %]
+                    </p>
+                    <p>
+                        [% l("The company holds customer funds in separate bank accounts to the operational accounts which would not, in the event of insolvency, form part of the Company's assets. This meets the Gambling Commission's requirements for the segregation of customer
```
Comment (jy-deriv) [inline src/templates/cashier/deposit_withdraw.html.tt]: remove all mention of Gambling Commission

File: src/templates/cashier/deposit_withdraw.html.tt [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    <div id="cashier_deposit" class="container">
+        <div class="row">
+            <div class="col-sm">
+                <h1 id="cashier_title" style="text-transform: capitalize"></h1>
+            </div>
+        </div>
+
+        <div class="row">
+            <div id="error_msg" class="col-xs-12 notice-msg center-text hidden"></div>
+            <div id="ukgc_funds_protection" class="hidden">
```
Comment (ashkanx) [inline src/templates/cashier/deposit_withdraw.html.tt]: @kellybinary   ukgc_funds_protection is for MX accounts. you don't need to handle this in champion-fx. remind me to explain this tomorrow.

File: src/javascript/pages/cashier/deposit_withdraw.js [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+                        error_msg.html('Your account is not fully authenticated.');
+                        break;
+                    case 'ASK_FINANCIAL_RISK_APPROVAL':
+                        error_msg
+                            .html('Financial Risk approval is required. Please contact <a href="[_1]">customer support</a> for more information.'
+                                .replace('[_1]', url_for('/contact')));
+                        break;
+                    case 'ASK_AGE_VERIFICATION':
+                        error_msg
+                            .html('Your account needs age verification. Please contact <a href="[_1]">customer support</a> for more information.'
```
Comment (ashkanx) [inline src/javascript/pages/cashier/deposit_withdraw.js]: Same here. age verifications is not required

File: src/javascript/pages/cashier/deposit_withdraw.js [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+                                .replace('[_1]', url_for('/contact')));
+                        break;
+                    case 'ASK_CURRENCY': // set account currency to USD if not set // TODO: remove this after currency set by default in backend
```
Comment (ashkanx) [inline src/javascript/pages/cashier/deposit_withdraw.js]: add this to a Trello card. we need to remove this when back-end fix is ready

**Review by ashkanx — CHANGES_REQUESTED** [review]

File: src/javascript/pages/cashier/deposit_withdraw.js (hunk 4 above)
Comment (kelly-binary) [reply src/javascript/pages/cashier/deposit_withdraw.js]: ok, this is listed as Todo in the card

**Review by mohammad-deriv — APPROVED** [review]

**Review by ashkanx — APPROVED** [review]

**Pull request merged**